src/main.rs

```diff
@@ -90,6 +90,11 @@ fn main() -> Result<(), TmsError> {
         set_up_tmux_env(found_repo, &repo_short_name)?;
     }
 
+    if !is_in_tmux_session() {
+        execute_tmux_command(&format!("tmux attach -t {repo_short_name}"));
+        return Ok(());
+    }
+
     let result = execute_tmux_command(&format!("tmux switch-client -t {repo_short_name}"));
     if !result.status.success() {
```

Comment:
I would have liked to remove this because it seems unnecessary, but I don't know if there was a specific reason for it.